src/components/radio-button-group/__tests__/RadioButtonGroup-test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
Let's keep in mind to create a test utils file in the near future to not access the render function directly, we can have the userEvent setup done automatically as part of it too.